src/concrete/ml/torch/hybrid_model.py

```diff
@@ -91,30 +191,97 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 
         Returns:
             (torch.Tensor): The output tensor.
+
+        Raises:
+            ValueError: if fhe_mode is not supported
         """
-        if self.fhe_local_mode != "disable":
-            # for mypy
-            assert self.private_module is not None
+        # - disable: torch module
```

Comment:
maybe something to put in the docstring instead of comments (nicer I think)

Reply:
This is more to have a quick reminder close to where this is used in the code but I can add it to the docstring indeed.